lib/composers/thenComposer.js

```diff
@@ -18,6 +19,16 @@ const makeThenComposer = (getSocketMessage, callback) => makeMethodComposer(
 			} else {
 				const socketMessage = getSocketMessage(data);
 
+				if (['query', 'testLine', 'eval'].includes(socketMessage.type)) {
```

Comment:
This expression `['query', 'testLine', 'eval'].includes`  is also used in websockets module, could be extracted as function.

Reply:
I don't like the idea of thenComposer knowing about socket message types. We are mixing two different concerns here.
Let's keep code clean and instead provide to `makeThenComposer` a `beforeSend` callback. This way each chain can decide if it wants to log a message and what message should it be. As a bonus, there would be no need to pass `toString` and `methodToString` here.